client/src/modules/general-ledger/general-ledger.ctrl.js

```diff
@@ -330,10 +331,28 @@ function GeneralLedgerController(
     });
   }
 
+  // specify if titles accounts should be hidden
+  function toggleHideTitlesAccounts() {
+    vm.hideTitlesAccounts = !vm.hideTitlesAccounts;
+    hideTitles();
+  }
+
+  // Hide when possible title accounts
+  function hideTitles() {
+    if (vm.hideTitlesAccounts) {
+      vm.gridOptions.data = vm.accounts.filter(t => {
+        return t.type_id !== 6;
```

Comment:
The `6` is the ID of the title account type, but that is unclear.  Thankfully, we have put this mapping in [`bhConstants`](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/js/constants/bhConstants.js#L16).  Please use the following:
```js
vm.gridOptions.data = vm.accounts.filter(account => account.type_id !== bhConstants.accounts.TITLE);
```

Reply:
You could actually make the `filter()` function into its own function for a performance boost:

```js
const isNotTitleAccount = (account) => account.type_id !== bhConstants.accounts.TITLE;

function hideTitles() {
  // ... etc ...
  vm.gridOptions.data = vm.accounts.filter(isNotTitleAccount);

}
```

The advantage here is that we don't make the function `isNotTitleAccount` every time `hideTitles()` is called.